src/config/dicom/manualBlacklist.ts

```diff
@@ -0,0 +1,373 @@
+  // 'SliceThickness',
+  // 'PixelSpacing',
+  // 'ImageOrientationPatient', (geometric data, not identifying)
+  // 'WindowCenter',
+  // 'WindowWidth',
+  // 'RescaleIntercept',
+  // 'RescaleSlope',
+
+  // === Protocol Names (Standardized) ===
+  // Many protocol names are standardized and don't contain patient info
+  // 'ProtocolName', (often standardized, but can contain custom text - keep monitoring)
+  // 'SeriesDescription', (often standardized, handled by cleanDescriptors options)
+
+  // === Sequence Attributes (Structured Data) ===
+  // Most sequence attributes contain structured data rather than free text
+  // 'ReferencedImageSequence',
+  // 'SourceImageSequence',
+  // 'DerivationImageSequence',
+  // 'ConceptNameCodeSequence',
+  // 'MeasurementUnitsCodeSequence',
+
+  // === Numeric IDs and Indices ===
+  // These are typically numeric identifiers without patient information
+  // 'InstanceNumber',
+  // 'SeriesNumber',
+  // 'AcquisitionNumber',
+  // 'ImageIndex',
+  // 'FrameNumber',
+
+  // === Standard Enumerated Values ===
+  // These contain predefined enumerated values, not free text
+  // 'Modality',
+  // 'PhotometricInterpretation',
+  // 'PlanarConfiguration',
+  // 'SamplesPerPixel',
+  // 'BitsAllocated',
+  // 'BitsStored',
+
+  // === Network/Communication (Technical) ===
+  // Technical network parameters, not patient identifying
+  // 'NetworkID', (if it's just network infrastructure)
+  // 'Port',
+  // 'HostName', (depends on naming convention - could be identifying)
+
+  // === Geometric/Spatial Data ===
+  // Pure geometric data without patient context
+  // 'ImagePositionPatient', (spatial coordinates, not identifying)
+  // 'SliceLocation',
+  // 'SpacingBetweenSlices',
+  // 'TableHeight',
+
+  // === Color/Display Parameters ===
+  // Technical display parameters
+  // 'RedPaletteColorLookupTableData',
+  // 'GreenPaletteColorLookupTableData',
+  // 'BluePaletteColorLookupTableData',
+
+  // ========================================================================
+  // EXCLUSION CRITERIA SUMMARY:
+  //
+  // Attributes are excluded from the blacklist if they:
+  // 1. Are UIDs (handled by special UID processing logic)
+  // 2. Contain only technical/equipment parameters (non-patient identifying)
+  // 3. Are standardized coded values (structured, not free text)
+  // 4. Are pure numeric measurements without context
+  // 5. Are already handled by existing PS3.15E1.1 rules
+  // 6. Are sequence attributes with structured data only
+  //
+  // When in doubt, err on the side of privacy - include in blacklist
+  // ========================================================================
+
+  // ========================================================================
+  // BLACKLISTED ATTRIBUTES (high risk for containing identifying information)
+  // ========================================================================
+
+  'ASLContext',
+  'ASLCrusherDescription',
```

Comment:
I suggest to remove all "Description" and "Comment" fields from this manual blacklist for consistency, as we're handling Descriptors in a consistent way already (per an actual option laid out in the 3.15 standard):
See `deidentifyPS315E.ts:370`